Kconfig.nrf

```diff
@@ -109,6 +109,17 @@ config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
 config GETOPT
 	default n
 
+# Hack to be able to run tests on the flpr core
```

Comment:
Is there some kconfig we could check here to ensure that it is a test that is being built? Should we do this in `test/Kconfig` instead of here?

Reply:
I suppose it's also used to test samples, so I don't think we can rely on a kconfig.